src/Controller/ExportController.php

```diff
@@ -136,6 +135,17 @@ private function getLang( Request $request ): string {
 		return strtolower( $lang );
 	}
 
+	/**
+	 * Get the format from the request.
+	 */
+	private function getFormat( Request $request ): string {
+		$format = $request->get( 'format' );
+		if ( !$format ) {
+			$format = 'epub-3';
```

Comment:
I want to mention that `epub-3` is a different default than before. Making sure this is on purpose. 
Nit: Maybe update the method documentation to mention the default if `format` is not specified?

Reply:
It's on purpose; they're synonymous. Although that's not very obvious I guess!

Docblock updated.